ibc/ibc_handle_event_test.go

```diff
@@ -0,0 +1,220 @@
+package ibc
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pokt-network/pocket/persistence/kvstore"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHandleEvent_FlushCaches(t *testing.T) {
+	mgr, _, _, _, ibcMod := prepareEnvironment(t, 1, 0, 0, 0)
+	tmpDir := mgr.GetConfig().IBC.StoresDir + "/caches"
```

Comment:
nit: prefer using the `path/filepath` over hardcoding os-dependent path delimiters (`/`: *nix, `\` windows)

```suggestion
	tmpDir := filepath.Join(mgr.GetConfig().IBC.StoresDir, "caches")
```